pom.xml

```diff
@@ -225,12 +225,11 @@
 
     <microprofile.jwt.version>2.0</microprofile.jwt.version>
     <microprofile.jwt.tck.version>2.0</microprofile.jwt.tck.version>
-    <microprofile.jwt.impl.version>${project.version}</microprofile.jwt.impl.version>
+    <microprofile.jwt.impl.version>4.0.0</microprofile.jwt.impl.version>
```

Comment:
I don't think this is what we want.
We have our own JWT implementation with more features than SmallRye so we use our own for this one.

Reply:
rolledback this change.